docs/index.md

```diff
@@ -0,0 +1,157 @@
+
+      - name: RESTful APIs
+        description: A rich API allows programmatic control over users, data, annotations, and algorithms, enabling automation of DSA tasks and integration with other tools and platforms.
+        icon: assets/img/feature-icons/icon-restful_apis.svg
+
+      - name: Visualization and Annotation
+        description: An optimized user interface provides fluid exploration of large whole-slide images and tools for efficient generation of image markups.
+        icon: assets/img/feature-icons/icon-visualization_annotation.svg
+
+      - name: Execution Engine
+        description: Girder provides distributed execution and monitoring of algorithm and analytics jobs.
+        icon: assets/img/feature-icons/icon-execution_engine.svg
+
+      - name: Broad Support for Histology Image Formats
+        description: A wide variety of tiled image formats are supported, including tiff, svs, and jp2. Images can be retiled automatically as needed for processing algorithms. Additional formats can be added with a pluggable Python interface.
+        icon: assets/img/feature-icons/icon-history_image_formats-primary.svg
+
+  # HistomicsTK
+  - title: HistomicsTK
+    description: HistomicsTK is a Python toolkit based on image processing, for quantitative analysis of whole-slide digital pathology images.
+    docs_url: google.com
+    github_url: github.com
+    screenshot: assets/img/histomicstk_screenshot.png
+    features:
+      - name: Preprocessing and Format Support
+        description: HistomicsTK provides color normalization and deconvolution operations to improve the robustness of analytic pipelines. The Large Image plugin gives programmers a convenient interface for reading from whole-slide imaging formats.
+        icon: assets/img/feature-icons/icon-preprocessing.svg
+
+      - name: Object Detection and Segmentation
+        description: HistomicsTK contains a number of classical image analysis and machine-learning based algorithms for object detection and segmentation of subcellular structures and tissues.
+        icon: assets/img/feature-icons/icon-object_detection.svg
+
+      - name: Feature Extraction and Predictive Modeling
+        description: Object and patch-level features describing shape, texture, and color can be used to build machine-learning models.
+        icon: assets/img/feature-icons/icon-feature_extraction.svg
+
+      - name: Extensibility
+        description: Users can integrate their custom algorithms through a containerization process that auto-generates DSA user-interfaces.
+        icon: assets/img/feature-icons/icon-extensibility.svg
+
+      - name: Broad Support for Histology Image Formats
+        description: A wide variety of tiled image formats are supported, including tiff, svs, and jp2. Images can be retiled automatically as needed for processing algorithms. Additional formats can be added with a pluggable Python interface.
```

Comment:
Let's change this side of the the image support format to read "The same wide variety of histology images that can be viewed can be used with any processing algorithms.  Sub-images can be processed at custom tile sizes and magnifications as needed."

Reply:
done